### DISCUSSION TITLE
How to deploy only particular project inside monorepo

### DISCUSSION THREAD

**Discussion by sankaran85** [vercel/vercel#5662] [Help]
Hi Team,

   I have mutiple nextjs app inside a monorepo. When i push code to one app , vercel build and deploys , all other apps as well. Is there is a way to build only one particular app and deploy it in vercel. Please guide me on this.

Thanks
Sankaran

**Comment by cromosom** [comment]
I use a bash script at "Ignored build step" which looks for certain triggers in my commit messages but I'm not very happy with my solution because it causes the other projects deploys to cancel. To really skip them would be a better handling of my pipelines.

**Reply by davidobodo** [reply]
hello @cromosom, thank you for this suggestion as I am currently trying to also execute this. 

Please would it be possible for you to share the bash script and give more explanation on how it works. Would appreciate the help. 
Thank you
